spring-cloud/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java

```diff
@@ -117,12 +118,25 @@ public void init() {
 		
 		logger.info("Solace client initializing and using SolaceMessagingInfo: " + solaceMessagingServiceInfo);
 
+		String host = solaceMessagingServiceInfo.getSmfHost();
+		
 		final JCSMPProperties properties = new JCSMPProperties();
-		properties.setProperty(JCSMPProperties.HOST, solaceMessagingServiceInfo.getSmfHost());
+		properties.setProperty(JCSMPProperties.HOST, host);
 		properties.setProperty(JCSMPProperties.VPN_NAME, solaceMessagingServiceInfo.getMsgVpnName());
 		properties.setProperty(JCSMPProperties.USERNAME, solaceMessagingServiceInfo.getClientUsername());
 		properties.setProperty(JCSMPProperties.PASSWORD, solaceMessagingServiceInfo.getClientPassword());
 		
+		// If using High Availability, the host property will be a comma-separated list of two hosts.
+        if (host.contains(",")) {
+        	// Recommended values for High Availability automatic reconnects.
```

Comment:
Can we add a query to the SolaceMessagingServiceInfo to ask if it HA. Users shouldn't have to parse for a ","